gradle.properties

```diff
@@ -79,7 +79,7 @@ ballerinaCommandVersion=1.3.15
 graphqlVersion=0.7.1-20230816-004300-12ef416
 
 # Protoc Tool
-protocToolVersion=0.1.2
+protocToolVersion=0.1.3-20230713-142600-0648ca1
```

Comment:
shouldnt we bump this to a new minor version with java17 change?

Reply:
Yes, once @gabilang send the migration PR will bump to the next minor version.

Reply:
Can we go ahead and merge this PR until protocTool is done?

Reply:
+1